compile.go

```diff
@@ -787,31 +788,107 @@ func ToQueryTime(value values.Value) (Time, error) {
 	}
 }
 
-var builtinImporter = &importer{
-	Packages: make(map[string]interpreter.Package),
+type pkg struct {
```

Comment:
How does this type differ from the `interpreter.packageObject` type?

The stdlib uses a map of `*pkg` types while the `Interpreter.Package()` method returns an `*interpreter.packageObject` type. Why do we need both types? It seems that only use of `Interpreter.Package` is to get at the side effects as for testing.